ray-on-gke/cloudbuild.yaml

```diff
@@ -1,17 +1,71 @@
 steps:
   - id: 'tf validate platform'
-    name: 'hashicorp/terraform:1.0.0'
+    name: 'gcr.io/$PROJECT_ID/terraform'
```

Comment:
Also, where is this image defined and do we always want `latest`?

Reply:
I think this image would only useful to run our cloudbuild. 
This image is to install normal terraform to cloud sdk. 
```
FROM gcr.io/google.com/cloudsdktool/cloud-sdk:latest

ARG VERSION=1.0.0

RUN apt-get install jq unzip -y

RUN curl https://releases.hashicorp.com/terraform/${VERSION}/terraform_${VERSION}_linux_amd64.zip -o terraform.zip
RUN unzip terraform.zip && rm terraform.zip
RUN mv terraform /usr/bin
```

The reason why we need to use this image instead of official terraform image is that
this step need both terraform and google-cloud-sdk-gke-gcloud-auth-plugin

Reply:
I am also looking for a good place to store this dockerfile